### PR TITLE
Changed to use shared RBAC service from api common

### PR DESCRIPTION
**Pull request by hsong-rh**

### CODE REVIEW

File: app/controllers/api/v1x0/mixins/rbac_mixin.rb [hunk 1]
```diff
@@ -151,8 +151,8 @@ def owner_stage_ids(request_ids)
 
         # The accessible workflow ids for approver
         def workflow_ids
-          approval_access = RBAC::Access.new('workflows', 'approve').process
-          approval_access.send(:generate_ids)
+          approval_access = Insights::API::Common::RBAC::Access.new('workflows', 'approve').process
+          approval_access.send(:ids)
```
Comment (bzwei) [inline app/controllers/api/v1x0/mixins/rbac_mixin.rb]: should not have called this private method

Comment (hsong-rh) [reply app/controllers/api/v1x0/mixins/rbac_mixin.rb]: Currently the method `ids` is defined as `private` in insights-api-common. We can change it later to allow us to call it directly here.

Comment (bzwei) [reply app/controllers/api/v1x0/mixins/rbac_mixin.rb]: `id_list` method is sufficient. It calls `ids` in the backend.

Comment (hsong-rh) [reply app/controllers/api/v1x0/mixins/rbac_mixin.rb]: Just realized this line is extra and should remove it.

**Pull request merged**